projects/multibit/index.js

```diff
@@ -10,38 +10,40 @@ const config = {
   ethereum: { stakingPool: "0x2EDfFbc62C3dfFD2a8FbAE3cd83A986B5bbB5495", tokens: [ADDRESSES.null, ADDRESSES.ethereum.USDT] },
   bsc: {},
   polygon: {},
-}
+  base: {},
+  savm: {}, // see: https://svmscan.io/address/0x861E3c82Bc2753ea64aE5f962d993DF6853a6700
+  coreDAO: {}, // see: https://scan.coredao.org/address/0x861e3c82bc2753ea64ae5f962d993df6853a6700
+};
 module.exports = {
   methodology: `Tokens bridged via MultiBit are counted as TVL`,
   bitcoin: {
     tvl: sumTokensExport({
       owners: [
-        'bc1p6r6hx759e3ulvggvd9x3df0rqh27jz59nvfjd2fzmh3wqyt6walq82u38z', // hot wallet
-        'bc1pyyms2ssr0hagy5j50r5n689e6ye0626v3c98j5fw0jk6tz3vrgts7nt56g',  // cold wallet
-        'bc1qmcrpqanjnrw58y0fvq08fqchgxv5aylctew7vxlkalfns3rpedxsx4hxpu',  // cold wallet
+        "bc1p6r6hx759e3ulvggvd9x3df0rqh27jz59nvfjd2fzmh3wqyt6walq82u38z", // hot wallet
+        "bc1pyyms2ssr0hagy5j50r5n689e6ye0626v3c98j5fw0jk6tz3vrgts7nt56g", // cold wallet
+        "bc1qmcrpqanjnrw58y0fvq08fqchgxv5aylctew7vxlkalfns3rpedxsx4hxpu", // cold wallet
       ],
-      blacklistedTokens: ['MUBI', 'BSSB']
```

Comment:
why are you removing BSSB?

Reply:
Hi @g1nt0ki 
BSSB is deployed by the Bitstable team and is not part of the Multibit team.

Reply:
yes, but we have had this discussion in previous PRs and we clearly said we wont include it.

Reply:
Yes, it was discussed in a previous PR.

But this Token is not directly related to the project.
It is just one of the cross-chain assets, and users just bridge it. It is similar to other coins.

Reply:
because multibit and BSSB are closely linked. we do the same for other projects as well, like when abra team minted and deposited SPELL into sushiswap's bentobox, we excluded it from tvl.

Reply:
This is an important issue that our team is discussing.

Since BSSB is one of our main recommended cross-chain assets,
If blacklisted, the TVL of some cooperation chains will be 0.

Reply:
hi @pipi-multibit 
I checked what rest of the team thinks, sorry, they agree that we cant count BSSB